docs/changelog.txt

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+v4.1 (2022-06-15, iplweb)
```

Comment:
What's iplweb?

```suggestion
v4.1 (2022-06-15)
```

Reply:
That's [iplweb ](https://github.com/iplweb)! Changelog updated. I'll be happy to remove my fork as soon as we'll be able to develop dbtemplates via Jazzband.

Reply:
I see you got added to the project in https://github.com/jazzband/django-dbtemplates/issues/121#issuecomment-1206571843 👍 

Are there any remaining blockers to develop via Jazzband? (Other than this PR)

Reply:
I created a few forked projects because they were not maintained and instead of renaming them I decided to keep the "-iplweb" suffix. In some cases, like in https://pypi.org/project/django-password-policies-iplweb/ they became pretty popular. 

As I had no contact with Jazzband team for some time and I really needed dbtemplates functionality for my app, I did the same thing with it. You can see the remnants of my changes, probably I should have cleaned that up before committing. 

If you will find me a worthy partner, I suppose I'll bring some more projects to Jazzband with me in the future.